core/src/main/java/apoc/periodic/Periodic.java

```diff
@@ -157,25 +166,64 @@ public Stream<JobInfo> cancel(@Name("name") String name) {
     public Stream<JobInfo> submit(@Name("name") String name, @Name("statement") String statement, @Name(value = "params", defaultValue = "{}") Map<String,Object> config) {
         validateQuery(statement);
         Map<String,Object> params = (Map)config.getOrDefault("params", Collections.emptyMap());
-        JobInfo info = submit(name, () -> {
+
+        final Temporal atTime = (Temporal) (config.get("atTime"));
+        
+        final Runnable task = () -> {
             try {
                 db.executeTransactionally(statement, params);
-            } catch(Exception e) {
+            } catch (Exception e) {
                 log.warn("in background task via submit", e);
                 throw new RuntimeException(e);
             }
-        }, log);
+        };
+
+        JobInfo info = atTime != null
+                ? getJobInfo(name, atTime, task, ScheduleType.DEFAULT) 
+                : submit(name, task);
+        
         return Stream.of(info);
     }
 
+    private JobInfo getJobInfo(String name, Temporal atTime, Runnable task, ScheduleType scheduleType) {
+        if (atTime instanceof LocalDate) {
+            atTime = ((LocalDate) atTime).atStartOfDay();
+        }
+        final boolean isTime = atTime instanceof OffsetTime || atTime instanceof LocalTime;
+        Temporal now = isTime
+                ? LocalTime.now()
+                : LocalDateTime.now();
+         
+        final long secPerDay = DateUtils.MILLIS_PER_DAY / 1000L;
+        long delay = now.until(atTime, ChronoUnit.SECONDS);
+        if (isTime && delay < 0) {
+            // we consider the day after
+            delay = delay + secPerDay;
+        }
+        if (delay < 0) {
+            throw new RuntimeException(ERROR_DATE_BEFORE);
+        }
+        return schedule(name, task, delay, secPerDay, scheduleType);
+    }
+
     @Procedure(mode = Mode.WRITE)
-    @Description("apoc.periodic.repeat('name',statement,repeat-rate-in-seconds, config) submit a repeatedly-called background statement. Fourth parameter 'config' is optional and can contain 'params' entry for nested statement.")
-    public Stream<JobInfo> repeat(@Name("name") String name, @Name("statement") String statement, @Name("rate") long rate, @Name(value = "config", defaultValue = "{}") Map<String,Object> config ) {
+    @Description("apoc.periodic.repeat('name',statement, rateOrTime, config) submit a repeatedly-called background statement. Fourth parameter 'config' is optional and can contain 'params' entry for nested statement.")
```

Comment:
I'm a bit uncertain about this change. If it can be considered a breaking change of the procedure, we should probably deprecate the old one and add a new one instead. But as it is a long that has been renamed and turned into an object, it maybe is just less strict and thus fine. Do you know how these sort of changes has been handled in the past @JMHReif ?